src/drivers/drv_mixer.h

```diff
@@ -71,11 +71,6 @@
 /** reset (clear) the mixer configuration */
 #define MIXERIOCRESET		_MIXERIOC(1)
 
-/**
- * add a simple mixer in (struct mixer_simple_s *)arg
- */
-#define MIXERIOCADDSIMPLE	_MIXERIOC(2)
-
```

Comment:
How do you know it's unused? 

And if that's really the case, what about this file?
https://github.com/PX4/Firmware/blob/master/src/lib/mixer/mixer_simple.cpp